src/cmdlib.sh

```diff
@@ -278,14 +278,52 @@ runvm() {
     # include COSA in the image
     find /usr/lib/coreos-assembler/ -type f > "${vmpreparedir}/hostfiles"
 
+    # virtfs contains additional 9p mounts, when needed
```

Comment:
The 9p stuff today conflicts with the container being RHEL-based right?  (Or at least it conflicts with a RHEL kernel)

Reply:
For EL7, yes.